### ISSUE TITLE
[Win][tabs on bottom] Possible alignment bug? Probably DPI related, maybe...

### ISSUE THREAD

**Issue by ImSpecial** [Aris-t2/CustomCSSforFx#61]
So I just noticed this, so this might not be new new, when tabs are on bottom there is this little gap where the edge of the window is and tab meets, using compact / dark theme / no title bar. Seems to not make a difference with what ever combo density / light/dark / title bar combo I use.

Here is with @import url(./css/tabs/tabs_below_navigation_toolbar.css); enabled.
https://i.imgur.com/7Z9LlBi.png

Here is with that line commented out, also tested by just renaming the chrome folder (ie. make FF stock) I get the same thing.
https://i.imgur.com/A9ELwY7.png

Monitors are 1920x1200, use Windows 10 custom DPI scaling @ 125% (if this info is at all helpful).

**Comment by Aris-t2** [comment]
I think this is independent of HiDPI modes.
"Tabs not on top" modes set a 2px gap like on Firefox 4-28, CTR for Firefox 29-56. But I will remove that gap for default dark and light themes on next update.

**Comment by MarkRH** [comment]
Yeah, I guess there is that slight gap on the left.
![image](https://user-images.githubusercontent.com/8035962/32990227-6f27d7c2-cceb-11e7-895a-13c9f00d760e.png)

1920x1080 monitor w/ normal scaling. Not that big a deal though.

**Comment by Aris-t2** [comment]
Without a gap the area, where the tab touches windows left edge, looks weird.

![123](https://user-images.githubusercontent.com/6013569/32990833-b59aa028-cd30-11e7-9566-2728cd37d55a.png)

**Comment by Aris-t2** [comment]
I will remove the padding for compact dark/light themes and lw-themes. It will stay for the default theme.

**Comment by Aris-t2** [comment]
Fixed on 1.3.2 for dark/light/lw-themes

**Comment by ImSpecial** [comment]
👍 

Hate to be a negative Nancy here but I think with the latest change, that [greyish line is back](https://github.com/Aris-t2/CustomCSSforFx/issues/29#issuecomment-344854093)...

https://i.imgur.com/EBLGr7Z.png

**Comment by Aris-t2** [comment]
Indeed, I will upload a new file soon.

**Comment by ImSpecial** [comment]
Thanks for the speedy fix, I'm an idiot for forgetting to mention this previously, but even with your fix and I think your initial fix was in 1.2.9, there was/is still a tiny tiny line noticeable on black backgrounds pages.

v1.3.2.1 = Looks great on pretty much anything not Black
https://i.imgur.com/dfA8EQQ.png

v1.3.2.1 = On something Black
https://i.imgur.com/NWFbpJw.png / https://i.imgur.com/OOQCk4a.png

Honestly I don't even mind it, but I'm not sure if it's considered a bug by you, or maybe an intentional feature.

**Comment by Aris-t2** [comment]
It was never my intension to completely remove the border, just making it less intrusive. It is more present for you, because you have set toolbars background to black.

Remove the line with this code:

```css
#main-window[style*='--lwt-header-image:url("resource:///chrome/browser/content/browser/defaultthemes/compact.header.png");']:-moz-lwtheme-brighttext #TabsToolbar:-moz-lwtheme {
  border-bottom: 0px !important;
}
```
